cmdLine/src/main/scala/topgun/cmdline/ClassLoaderInfo.scala

```diff
@@ -0,0 +1,32 @@
+package topgun.cmdline
+
+
+import java.io.InputStream
+
+import org.objectweb.asm.ClassReader
+import org.objectweb.asm.tree.ClassNode
+
+import scala.collection.mutable
+
+class ClassLoaderInfo() {
+  var allClasses = new mutable.HashMap[String, ClassInfo]()
+
+  def lookup(clazzName: String, clazzLoader: ClassLoader): ClassInfo = {
+    val key = clazzName.replace(".", "/") + ".class"
+    allClasses.getOrElseUpdate(clazzName, searchForAndPutClassInfoInToMap(key, clazzLoader)): ClassInfo
+  }
+
+  def searchForAndPutClassInfoInToMap(key: String, clazzLoader: ClassLoader): ClassInfo = {
+    val iStream: InputStream = clazzLoader.getResourceAsStream(key)
```

Comment:
where is this stream closed?

Reply:
this wasn't closed fixed it in the commit.